video/out/gpu/video.c

```diff
@@ -2994,12 +3000,24 @@ static void pass_draw_to_screen(struct gl_video *p, struct ra_fbo fbo)
     // Adjust the overall gamma before drawing to screen
     if (p->user_gamma != 1) {
         gl_sc_uniform_f(p->sc, "user_gamma", p->user_gamma);
-        GLSL(color.rgb = clamp(color.rgb, 0.0, 1.0);)
+        // Experimental #HDR on #macOS        
+        // We only clamp RGB values when HDR is not enabled
+        if (p->opts.tone_map.curve != TONE_MAPPING_HDR_PASSTHROUGH && p->opts.tone_map.curve != TONE_MAPPING_HDR_SCALE)
+        {
+            GLSL(color.rgb = clamp(color.rgb, 0.0, 1.0);)
+        }
```

Comment:
Rather than hackily implementing it like this, we could probably just relax the `clamp(color.rgb, 0.0, 1.0);` to `color.rgb = max(color.rgb, 0.0)`. The clip to positive values is still needed because gamma functions are undefined for negative values.

Reply:
Yeah, I see. Sure!